### DISCUSSION TITLE
Replace overseerr docker container with jellyseerr image?

### DISCUSSION THREAD

**Discussion by famewolf** [seerr-team/seerr#650] [Q&A] [answered]
I currently have a  Docker (Kubernetes) image running on my truenas server......if I wanted to switch to Jellyseerr could I just replace the image pulled?   I'm guess there would be other changes but figured it never hurts to ask.

**Comment by banjo508** [comment, marked as answer]
Jellyseer is pretty simple to setup. Just redeploy.
